core/src/main/scripts/importer/validateStudies.py

```diff
@@ -131,11 +131,12 @@ def main(args):
             exit_status_study = 2
 
         # Check exit status and print result
-        if exit_status_study == 1 or exit_status_study == 2:
-            print '\x1b[0m' + "Result: " + '\x1b[31m' + possible_exit_status[exit_status_study] + '\x1b[0m'
-            validation_exit_status = 1  # When invalid check the exit status to one, for failing circleCI
+        exit_status_message = possible_exit_status.get(exit_status_study, 'Unknown status: {}'.format(exit_status_study))
+        if exit_status_study == 0 or exit_status_study == 3:
+            print '\x1b[0m' + "Result: %s" % exit_status_message
```

Comment:
Moving these ANSI colour codes into a little namespace, or reusing the existing one in [metaImport.py](https://github.com/cBioPortal/cbioportal/blob/v1.14.0/core/src/main/scripts/importer/metaImport.py#L22), would make this code a *lot* cleaner, if you're touching this code anyway. But again, if you feel it's disproportionate to the scope of this PR, this will do!